fdbcli/SetClassCommand.actor.cpp

```diff
@@ -77,18 +77,18 @@ ACTOR Future<bool> setProcessClass(Reference<IDatabase> db, KeyRef network_addre
 	loop {
 		tr->setOption(FDBTransactionOptions::SPECIAL_KEY_SPACE_ENABLE_WRITES);
 		try {
+			state ThreadFuture<Optional<Value>> result =
+			    tr->get(network_address.withPrefix(fdb_cli::processClassTypeSpecialKeyRange.begin));
+			Optional<Value> val = wait(safeThreadFutureToFuture(result));
+			if (!val.present()) {
+				printf("No matching addresses found\n");
+				return false;
+			}
 			tr->set(network_address.withPrefix(fdb_cli::processClassTypeSpecialKeyRange.begin), class_type);
 			wait(safeThreadFutureToFuture(tr->commit()));
 			return true;
 		} catch (Error& e) {
-			state Error err(e);
-			if (e.code() == error_code_special_keys_api_failure) {
-				std::string errorMsgStr = wait(fdb_cli::getSpecialKeysFailureErrorMessage(tr));
-				// error message already has \n at the end
-				fprintf(stderr, "%s", errorMsgStr.c_str());
-				return false;
-			}
-			wait(safeThreadFutureToFuture(tr->onError(err)));
+			wait(safeThreadFutureToFuture(tr->onError(e)));
```

Comment:
I see this removes the previous error message logging, was that intentional?

Reply:
@sfc-gh-ajbeamon oh, perhaps I misunderstood your previous comment re: `_I also notice that the second transaction specifically checks for a special keys API failure. I'm not sure if that's needed in this transaction, though combining into one transaction makes that question moot._`

I will fix to keep that check.